docs/examples/advanced-cue/kautoscale.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: core.oam.dev/v1alpha2
+kind: TraitDefinition
+metadata:
+  annotations:
+    definition.oam.dev/description: "Specify auto scale by annotation"
+  name: kautoscale
+spec:
+  appliesToWorkloads:
+    - webservice
```

Comment:
It only works with knative workload type

Reply:
we don't have knative in real example, I add an annotation